### PR TITLE
Inventry 

### PR DESCRIPTION
**Pull request by Ayukha**
Student Dashboard backend added

### CODE REVIEW

File: Proj_i02/settings.py [hunk 1]
```diff
@@ -102,16 +102,20 @@
 
 AUTH_PASSWORD_VALIDATORS = [
```
Comment (dhruvhacks) [inline Proj_i02/settings.py]: This change is part of [#13](https://github.com/ncs-jss/Proj_i02/pull/13)
Merge it in ```inventory``` branch to update this PR.

File: Proj_i02/urls.py [hunk 2]
```diff
@@ -21,7 +21,10 @@
 
 
 urlpatterns = [
-    url(r'^login/', include("accounts.urls")),
+
+    url(r'^', include("accounts.urls")),
+    url(r'^', include("inventory.urls")),
```
Comment (dhruvhacks) [inline Proj_i02/urls.py]: Have a separate URL for either `inventory.urls``` or ```accounts.urls```

File: accounts/views.py [hunk 3]
```diff
@@ -1,14 +1,115 @@
...
+
+            return render(request, "accounts/login.html")
+        if data.get('group') == "faculty" or data.get('group') == "hod":
+            print("Faculty login")
+            request.session["id"] = data.get('username')
+            request.session["group"] = data.get('group')
+
+            if FacultyProfile.objects.filter(
+                    username=data.get('username')).count() == 1:
+                print("already there")
```
Comment (dhruvhacks) [inline accounts/views.py]: Remove all debug statements.

File: accounts/views.py [hunk 4]
```diff
@@ -1,14 +1,115 @@
...
+        email = request.POST.get('email')
+        Phone = request.POST.get('phone')
+        StudentProfile.objects.create(
+            username=request.session['id'],
+            group=branch, Branch=branch,
+            email=email, phone=Phone)
+        return HttpResponse("succesful")
+
+
+class Facultyprofileupdate(View):
```
Comment (dhruvhacks) [inline accounts/views.py]: If anyone hits the URL for this view, ANYONE CAN CREATE A FACULTY PROFILE!
Provide the required authentication check here!!

File: inventory/admin.py [hunk 5]
```diff
@@ -50,4 +50,5 @@ class LocationAdmin(admin.ModelAdmin):
 
 
 admin.site.register(models.inventory)
+# admin.site.register(models.count_division)
```
Comment (dhruvhacks) [inline inventory/admin.py]: Un-comment this since it will be required to add any item in ```location```.

**Comment by dhruvhacks** [comment]
@Ayukha ??

**Comment by dhruvhacks** [comment]
@Ayukha As discussed earlier, you may remove the migration files and update this PR.
If you still feel to push the migration files then delete these migrations, create a single one and use it here.

**Comment by Ayukha** [comment]
@dhruvhacks  Please Review

**Pull request merged**